### ISSUE TITLE
Is this project still alive?

### ISSUE THREAD

**Issue by vaclavsvejcar** [typelead/eta#984]
Based on the fact that there's not much commit/release activity in last half year, I'm wondering if this project is still alive, under development, or has it been abandonned?

**Comment by GGXBoo** [comment]
I feel dead

**Comment by jneira** [comment]
Well, i want to think it is in hibernation, i hope it is temporarily.
I am afraid that the lack of open source contibutors is one of the main causes.

**Comment by ShalokShalom** [comment]
Most open source projects of this size are maintained by a small handfull of people, so luckily not many are needed.

Just someone who can commit to it full time.

I hope this happens, since Eta is an important piece of software.

**Comment by Ericson2314** [comment]
I feel like at this point the best thing would be to see this upstreamed into GHC after all. The issues about research vs pragmatics that were given as motivation for the fork are being addressed better than ever before in upstream GHC. There is also the new [Haskell Foundation](https://haskell.foundation/) to help ensure that momentum continues and in fact grows.

**Comment by ShalokShalom** [comment]
Concrete criticism and what holds them back is what I am most interested in.

**Comment by georgefst** [comment]
> I feel like at this point the best thing would be to see this upstreamed into GHC after all.

Indeed. Especially with [all the great work going on around cross-compilation support](https://gitlab.haskell.org/ghc/ghc/-/wikis/cross-compilation/roadmap), as motivated by the new JS and WASM backends.

Sadly, it would take someone highly motivated, and probably significant guidance from the original developers, and I just don't see any evidence it's likely.
